WrathCombo/Combos/PvP/DRGPVP.cs

```diff
@@ -79,6 +80,9 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
                         if (IsEnabled(CustomComboPreset.DRGPvP_WyrmwindThrust) && HasEffect(Buffs.FirstmindsFocus) && GetTargetDistance() >= GetOptionValue(Config.DRGPvP_Distance_Threshold))
                             return WyrmwindThrust;
                     }
+
+                    if (IsEnabled(CustomComboPreset.DRGPvP_ChaoticSpringExecute) && IsOffCooldown(ChaoticSpring) && EnemyHealthCurrentHp() <= 8000)
+                            return ChaoticSpring;
```

Comment:
Just some food for thought as a DRG player.
Chaotic Spring also has a healing component (and a strong one at that).
I personally would give it two conditions: target's health is low **OR** your health is low.
If you're feeling frisky, you can also add a `hasLifeOfTheDragon` condition to ensure boosted damage.

Reply:
![image](https://github.com/user-attachments/assets/c78ecc5f-ed9d-4762-93ec-73b1aa6a3f14)
I added and execute, but the sustain for health section is still there higher up. and the execute is tied to the amount of damage it would do. 8k